src/Components/CollectionsRail/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+import { graphql, QueryRenderer } from "react-relay"
+
+import { CollectionsRailQuery } from "__generated__/CollectionsRailQuery.graphql"
+import { ContextConsumer } from "Artsy"
+import { CollectionsRailFragmentContainer as CollectionsRail } from "./CollectionsRail"
+
+interface Props {
+  showOnEditorial?: boolean
+}
+
+export class CollectionsRailContent extends React.Component<Props, null> {
```

Comment:
This can be a `React.SFC`, and if no state, don't need to pass `null` as an interface:
```js
React.SFC<Props>
```